src/_locales/en/messages.json

```diff
@@ -138,10 +138,10 @@
     "message": "Find shortcuts for this extension by going to <a>extension settings.</a>"
   },
   "welcomePageTryFirefox": {
-    "message": "Want to hop back to Firefox? <a>Try our Chromium extension.</a>"
+    "message": "Want to hop back to Firefox from another browser? <a>Try our Chrome extension.</a>"
   },
   "welcomePageTryChromium": {
```

Comment:
Since this string is changing and will require an updated translation, we will need to update the string ID to trigger retranslations in Pontoon.

Reply:
Do you mean update the ID as in give this string here a new ID, allowing pontoon to recognize it as a new translation?

Or do you mean some other thing has to happen within pontoon to replace the old translations?

Reply:
The former, any string with changes (apart from minor things like capitalization) need a new ID to recognize it as a new string. (If you update an existing string without changing the ID, there is no triggering of retranslations, so translations would effectively be out of date.)